frontend/src/pages/document.tsx

```diff
@@ -17,15 +17,67 @@ import { WebvttExportModal } from '../editor/webvtt_export';
 import { showModal } from '../components/modal';
 import { Tooltip } from '../components/tooltip';
 import { SpeakerColorsProvider } from '../editor/speaker_colors';
+import { Input } from '../components/form';
+import { BiPencil } from 'react-icons/bi';
 
 const LazyDebugPanel = lazy(() =>
   import('../editor/debug_panel').then((module) => ({ default: module.DebugPanel })),
 );
 
+function DocumentTitle({ name, onChange }: { name: string; onChange: (newTitle: string) => void }) {
```

Comment:
Could you please center the buttons?

<img width="952" alt="Screenshot 2023-05-27 at 15 11 53" src="https://github.com/transcribee/transcribee/assets/6545097/6d351975-4cb5-4feb-9464-f81ad9dec654">